supply-chain/platform-engineering/how-to/installing-supply-chain/post-install-configuration.hbs.md

```diff
@@ -4,80 +4,86 @@
+After you have installed Tanzu Supply Chain, there are several steps you must complete to ensure
+you have a functioning installation. Use Namespace Provisioner to configure service accounts and permissions.
 
-## Namespace Provisioner
+## Things to configure
 
-The recommended way of configuring service accounts and permissions on Tanzu Application Platform is
-to use the Namespace Provisioner.
+- OCI Store Configuration
+  - Supply Chains persist data between stages by reading and writing to an OCI repository.  The location of the OCI repository is configured by a Kubernetes Secret named `oci-store` that exists within the developer namespace.
+  - Access to this repository is controlled by a tekton annotated secret which can have any name as long as it has the `tekton.dev/docker-0` annotation pointing to the OCI repository.
+- Permissions for Buildpacks Cluster Builders for buildpack-build component
+  - If you are planning to use the `buildpack-build` component to create images using TBS configured with `ClusterBuilders` you must add some additional permissions.
 
-```shell
-❯ kubectl create namespace my-supply-chains
-❯ kubectl label namespaces my-supply-chains apps.tanzu.vmware.com/tap-ns=""
+## Setup using Namespace Provisioner
+
+Recommended way is to use the Namespace provisioner for the entire setup:
+
+**Step 1**: is to create a `Secret` in the `tap-install` namespace that has the location and credentials for the `oci-store` as follows:
+
+```console
+cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: supply-chain-oci-store-credentials
+  namespace: tap-install
+type: Opaque
+stringData:
+  ocistore.yaml: |
+    tanzusupplychain:
+      ocistore:
+        username: <registry-username>
+        password: <registry-password>
+        server: <registry-server>
+        repository: <registry-repo>
+EOF
 ```
 
-If Namespace Provisioner is not enabled in your environment, add the following section in `tap-values.yaml` to enable it:
+**Step 2**: Configure Namespace Provisioner to use the accelerator sample that will help with creation of all required resources for configuring `oci-store` as well as `buildpack-build` permissions.
 
-```yaml
+```console
 namespace_provisioner:
-  controller: true
+  additional_sources:
+  - git:
+      ref: origin/main
+      subPath: ns-provisioner-samples/tanzu-supply-chain
+      url: https://github.com/vmware-tanzu/application-accelerator-samples.git
+  import_data_values_secrets:
+  - name: supply-chain-oci-store-credentials
+    namespace: tap-install
+    create_export: true
   default_parameters:
     supply_chain_service_account:
       secrets:
-        # List your git & docker secrets here
-        - git-secret
-        - my-oci-store-credentials # this secret is described below
+      - oci-store-credentials
 ```
 
-## OCI Store
+## Create Developer Namespaces
```

Comment:
Is create Developer Namespaces a consecutive step after  the previous step to - configure Namespace Provisioner to use the accelerator sample?
Or is it maybe a prereq step for some other procedure?